app/controllers/groups_controller.rb

```diff
@@ -323,8 +323,8 @@ def add_members
         'usernames or emails must be present'
       )
     end
-
-    if (usernames = group.users.where(id: users.map(&:id)).pluck(:username)).present?
+    usernames = group.users.where(id: users.map(&:id)).pluck(:username)
```

Comment:
This is a nitpick but I think it will make things clearer; maybe change this variable from `usernames` to `usernames_already_in_group`?